PSReadLine/KeyBindings.cs

```diff
@@ -339,6 +339,9 @@ void SetDefaultEmacsBindings()
                 { Keys.AltH,                   MakeKeyHandler(ShowParameterHelp,         "ShowParameterHelp") },
                 { Keys.F1,                     MakeKeyHandler(ShowCommandHelp,           "ShowCommandHelp") },
                 { Keys.F2,                     MakeKeyHandler(SwitchPredictionView,      "SwitchPredictionView") },
+                { Keys.AltU,                   MakeKeyHandler(UpcaseWord,                "UpcaseWord") },
+                { Keys.AltL,                   MakeKeyHandler(DowncaseWord,              "DowncaseWord") },
+                { Keys.AltC,                   MakeKeyHandler(CapitalizeWord,            "CapitalizeWord") },
```

Comment:
Please add tests for these changes. See examples in https://github.com/PowerShell/PSReadLine/blob/master/test/BasicEditingTest.cs

Reply:
The tests will take some time as I'm not familiar with them. But the rest I will address quickly. Thanks for the reviews.

One question. Are these keybindings for letting Esc key behave like a pressed-down Alt key? If so, should I add my three functions to this list?

https://github.com/PowerShell/PSReadLine/blob/bbbe43bde346c443e236ac39fff1103cd3b2ccd0/PSReadLine/KeyBindings.cs#L367-L378

Reply:
Both Alt and Esc versions of those keys work in macOS bash. Can't confirm/deny about its behavior on .NET Core, though.

I added the Esc versions keybindings as well. Easy enough to get rid off if someone files a bug. So why not.